metricbeat/module/elasticsearch/elasticsearch_integration_test.go

```diff
@@ -180,6 +180,14 @@ func enableTrialLicense(host string) error {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode != 200 {
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		return fmt.Errorf("could not enable trial license, response = %v", string(body))
```

Comment:
should we use logp.Error instead?

Reply:
I'm not sure how that would work here, since we're _returning_ an error message not emitting one to a logger?

Reply:
oh sorry yeah I didn't realize this is a part of `return`.